go.sum

```diff
@@ -748,6 +748,7 @@ github.com/jsonnet-bundler/jsonnet-bundler v0.1.1-0.20190930114713-10e24cb86976/
 github.com/jsonnet-bundler/jsonnet-bundler v0.2.0/go.mod h1:/by7P/OoohkI3q4CgSFqcoFsVY+IaNbzOVDknEsKDeU=
 github.com/jstemmer/go-junit-report v0.0.0-20190106144839-af01ea7f8024/go.mod h1:6v2b51hI/fHJwM22ozAgKL4VKDeJcHhJFhtBdhmNjmU=
 github.com/jstemmer/go-junit-report v0.9.1/go.mod h1:Brl9GWCQeLvo8nXZwPNNblvFj/XSXhF0NWZEnDohbsk=
+github.com/jteeuwen/go-bindata v0.0.0-20151023091102-a0ff2567cfb7 h1:fNOeTlrOg7VMXyGgvnHoqw6owD7hd3mKSlivApPJMzI=
```

Comment:
I'm not 100%, but I've done a bit of research into this and I think it's because a new module version of `go-bindata` was used to build the system probe on the agent dev box. If I remove this line and build again, then this line is always added to `go.sum`. 

Looking at the go modules documentation, the "go.sum contains the expected cryptographic checksums of the content of specific module versions," so I don't think it will hurt anything to include this value. Documentation here:

- https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file

Worst case, it is just removed at a later point in time.

Reply:
I'm pretty sure this is because `inv -e system-probe.build` downloads its own version of go-bindata. 

This should be fixed by https://github.com/DataDog/datadog-agent/pull/5792   .